src/components/Attachments/AttachmentCarousel/attachmentCarouselPropTypes.js

```diff
@@ -20,11 +21,23 @@ const propTypes = {
 
     /** The report currently being looked at */
     report: reportPropTypes.isRequired,
+
+    /** The parent of `report` */
+    parentReport: reportPropTypes,
+
+    /** The report actions of the parent report */
+    parentReportActions: PropTypes.shape(reportActionPropTypes),
```

Comment:
This kind of mistake is easy to miss
```suggestion
    parentReportActions: PropTypes.objectOf(PropTypes.shape(reportActionPropTypes)),
```

Reply:
Yes, thank you. I went back and forth on `parentReportAction` and `parentReportActions` and forgot to update this.